src/file.coffee

```
@@ -77,12 +74,12 @@ fileContent = (filename, options = {}) ->
     { data, time: Date.now(), source: filename }
 
   load = partial fromPromiseFunction, ->
-    debug 'readFile %s', filename
-    readFile(filename, 'utf8')
+    debug 'readFileAsync %s', filename
+    readFileAsync(filename, 'utf8')
       .tap loaded
       .then parse
       .then wrap
-      .catch isMissingError, returnDefault
+      .catch returnDefault
```

Doesn't this change it from "only swallow missing files" to "swallow all errors"? E.g. it's just removing the condition from the catch, turning it into a catch-all?